docs/reference/security/authorization/managing-roles.asciidoc

```diff
@@ -209,6 +214,35 @@ The following describes the structure of a remote indices permissions entry:
     restricted indices, you must set this field to `true` (default is `false`), and then the
     `names` field will cover the restricted indices as well.
 
+[[roles-remote-cluster-priv]]
+==== Remote cluster privileges
+
+For <<remote-clusters-api-key,remote clusters configured with the API key based model>>, remote cluster privileges
+can be used to specify additional cluster privileges for matching remote clusters.
+
+NOTE: Remote cluster privileges are effective for remote clusters configured with the API key based model.
+They have no effect for remote clusters configured with the certificate based model.
+
+The following describes the structure of a remote cluster permissions entry:
+
+[source,js]
+-------
+{
+  "clusters": [ ... ], <1>
+  "privileges": [ ... ] <2>
+}
+-------
+// NOTCONSOLE
+<1> A list of remote cluster aliases. It supports literal strings as well as
+<<api-multi-index,wildcards>> and <<regexp-syntax,regular expressions>>.
+This field is required.
+<2> The cluster level privileges for the remote cluster. The allowed values here are a subset of the
+<<privileges-list-cluster,cluster privileges>>. This field is required.
+
+The only privilege that can be granted for remote clusters is `monitor_enrich` for
+versions 8.15.0 and later. `monitor_enrich` is requires to allow users to use the `ENRICH` keyword for
+ES|QL queries across clusters.
```

Comment:
```suggestion
The `monitor_enrich` privilege for remote clusters was introduced in version
8.15.0. Currently, this is the only privilege available for remote clusters and
is required to enable users to use the `ENRICH` keyword in ES|QL queries across
clusters.
```

Reply:
I think this might be a little clearer, let me know if I've understood correctly :)